student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] **[OPTIONAL]** I suggest you pass arguments to the `super` call like this `super(age, name, parent_permission)` so that you can specify the arguments you want to get from the parent constructor if later more arguments are added to the parent's constructor but you only want to get these ones. You can do the same in the `Teacher` class

```suggestion
  def initialize(classroom, age, name = "Unknown", parent_permission: true)
    super(age, name, parent_permission)
```